test/jdk/com/sun/net/httpserver/simpleserver/DocRootDirPermissionsWinTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @summary Tests file permission checks during the creation of a `FileServerHandler`
+ * @requires (os.family == "windows")
+ * @library /test/lib
+ * @build jdk.test.lib.net.URIBuilder
+ * @run main/manual/othervm -ea DocRootDirPermissionsWinTest true
+ * @run main/manual/othervm -ea DocRootDirPermissionsWinTest false
+ */
+
+public class DocRootDirPermissionsWinTest extends DocRootDirPermissionsTest{
```

Comment:
Hello Serhiy, the test doesn't necessarily have to extend from that other test and we could probably just make `DocRootDirPermissionsTest.run(String[] args)` a static method and call that from within the main() method of this `DocRootDirPermissionsWinTest`. But I see some work was done to make those methods instance methods instead of static which means that you might have to revert that. So I think it's fine in the current form and I see Daniel has reviewed the PR and you have run this for tier1 and tier2. I'll go ahead and sponsor this now.